src/solc_0.6/EstateSale/EstateSaleWithFee.sol

```diff
@@ -93,7 +93,7 @@ contract EstateSaleWithFee is MetaTransactionReceiver, ReferralValidator {
         uint256[] calldata values
     ) external {
         require(msg.sender == _admin, "NOT_AUTHORIZED");
-        require(block.timestamp > _expiryTime, "SALE_NOT_OVER");
+        // require(block.timestamp > _expiryTime, "SALE_NOT_OVER"); // removed to recover in case of misconfigured sales
```

Comment:
Will this be reinstated for Mainnet?

Reply:
I don't think it is necessary as we can already stop sale by disabling minter, so I think letting us recover from misconfigured landsale is better